### PR TITLE
Add panel from Rust for the Web London

### PR DESCRIPTION
**Pull request by emmasofiah2o**
- Add panel
- Add visual

Screenshot 2024-03-20 at 10 13 07

### CODE REVIEW

File: src/appearances/2024-02-07-rust-in-production-why-how.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+image: "/assets/images/talks/2024-02-07-rustweb-london/thumbnail-panel.jpg"
+url: https://youtu.be/ZRBayed8cLw
+media: video
+channel: rustweb-london
+---
+
+Moderator Luca Palmieri (Principal Engineering Consultant at Mainmatter) and
+panelists ​Edward Wright (Lead GIS Engineer at Vortexa), ​Nodar Daneliya
+(Founder and CEO of Shuttle), and ​James Cole (Arwen.ai), discussed the reasons
+for choosing Rust and how to use it for production at our Rust for the Web
```
Comment (marcoow) [inline src/appearances/2024-02-07-rust-in-production-why-how.md]:
```suggestion
for choosing Rust and how they use it in production at our Rust for the Web
```

**Review by marcoow — CHANGES_REQUESTED** [review]
slight but important difference

File: src/appearances/2024-02-07-rust-in-production-why-how.md (hunk 1 above)
Comment (emmasofiah2o) [reply src/appearances/2024-02-07-rust-in-production-why-how.md]: Should I change the YouTube description as well, then?

Comment (marcoow) [reply src/appearances/2024-02-07-rust-in-production-why-how.md]: yes please

File: src/appearances/2024-02-07-rust-in-production-why-how.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+url: https://youtu.be/ZRBayed8cLw
+media: video
+channel: rustweb-london
+---
+
+Moderator Luca Palmieri (Principal Engineering Consultant at Mainmatter) and
+panelists ​Edward Wright (Lead GIS Engineer at Vortexa), ​Nodar Daneliya
+(Founder and CEO of Shuttle), and ​James Cole (Arwen.ai), discussed the reasons
+for choosing Rust and how they use it in production at our Rust for the Web
+Meetup. at our Rust for the Web meetup in London.
```
Comment (marcoow) [inline src/appearances/2024-02-07-rust-in-production-why-how.md]:
```suggestion
for choosing Rust and how they use it in production at our Rust for the Web
meetup in London.
```

**Review by marcoow — APPROVED** [review]

**Pull request merged**